### PR TITLE
Extended readme files

### PR DESCRIPTION
**Pull request by FelixWohlfrom**
This should help now also people that are not to familiar with nodejs and react
for a fast start.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
- [![](https://img.shields.io/badge/contribute-info-blue?style=flat-square)](https://github.com/wintergatan-community/virtual-mmx/blob/master/CONTRIBUTING.md)
- [![Discord](https://img.shields.io/badge/Discord-join-7289DA?logo=discord&style=flat-square)](https://discord.gg/rMK6DFT)
+[![](https://img.shields.io/github/issues-pr/wintergatan-community/virtual-mmx)](https://github.com/wintergatan-community/virtual-mmx/pulls)
+[![](https://img.shields.io/badge/contribute-info-blue?style=flat-square)](https://github.com/wintergatan-community/virtual-mmx/blob/master/CONTRIBUTING.md)
+[![Discord](https://img.shields.io/badge/Discord-join-7289DA?logo=discord&style=flat-square)](https://discord.gg/rMK6DFT)
 
-A WebApp to program songs on a virtual Marble Machine X
+## About
+
+A WebApp to program songs on a virtual Marble Machine X. Songs can be exported and stored on local disk.
```
Comment (mozi-h) [inline README.md:10]: Perhaps mention that future features can be seen on this board https://github.com/orgs/wintergatan-community/projects/1

File: README.md [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+
+Exported data is using the structure described in the [vmmx schema](https://github.com/wintergatan-community/vmmx-schema) github repository.
```
Comment (mozi-h) [inline README.md]: It's also the internal data structure – not just the exported JSON structure

**Review by mozi-h — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (FelixWohlfrom) [reply README.md:10]: Yes, that will also be helpful. Added.

File: README.md (hunk 2 above)
Comment (FelixWohlfrom) [reply README.md]: My intention was, that this information should be added to some "architecture" page, but for now we can add it also here :)

File: README.md [hunk 3]
```diff
@@ -1,8 +1,19 @@
...
 
-A WebApp to program songs on a virtual Marble Machine X
+## About
+
+A WebApp to program songs on a virtual Marble Machine X. Songs can be exported and stored on local disk.
+
+To view the future plans we have for this WebApp, please visit our [planning board](https://github.com/orgs/wintergatan-community/projects/1).
+
+Exported data is using the structure described in the [vmmx schema](https://github.com/wintergatan-community/vmmx-schema) github repository.
+This repository also contains the definition for our internal data structure.
```
Comment (mozi-h) [inline README.md]: It’s in the vmmx-schema repository. With my comment above, I intended for line 14 to be rephrased to something like „Internal data structure is described in ...“

**Review by mozi-h — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (FelixWohlfrom) [reply README.md]: Ah, now I got you. I hope it's better now :)

**Review by mozi-h — APPROVED** [review]
Neat 👍

**Pull request merged**